Jenkinsfile

```diff
@@ -1,7 +1,22 @@
-// Readme @ http://gitlab.iex.ec:30000/iexec/jenkins-library
-@Library('global-jenkins-library@1.0.7') _
+@Library('global-jenkins-library@1.6.0') _
 
-def nativeImage = buildSimpleDocker_v2(dockerfileDir: './docker', buildContext: '.',
-        dockerImageRepositoryName: 'tee-worker-post-compute', imageprivacy: 'public')
-sconeBuildAllTee(nativeImage: nativeImage, targetImageRepositoryName: 'tee-worker-post-compute',
-        sconifyArgsPath: './docker/sconify.args')
\ No newline at end of file
+String repositoryName = 'tee-worker-post-compute'
+
+buildJavaProject(
+        integrationTestsEnvVars: [],
+        shouldPublishJars: true,
+        shouldPublishDockerImages: true,
+        dockerfileDir: 'docker',
+        buildContext: '.',
+        dockerImageRepositoryName: repositoryName,
+        preProductionVisibility: 'docker.io',
+        productionVisibility: 'docker.io')
+
+node('docker') { //any node would be fine
```

Comment:
Any node with minimum specs?

Reply:
Only required for processing sh command line yes

Reply:
I mean we need a minimum set of specs to run TEE builds.

Reply:
Not here, `sconeBuildAllTee` already defines its own node specification (which we might upgrade btw)

Reply:
Ok thanks for the clarification.